jans-cache-refresh/src/main/java/io/jans/cacherefresh/service/AppInitializer.java

```diff
@@ -0,0 +1,300 @@
+import io.jans.orm.model.PersistenceConfiguration;
+import io.jans.orm.util.properties.FileConfiguration;
+import io.jans.service.PythonService;
+import io.jans.service.cdi.event.ApplicationInitialized;
+import io.jans.service.cdi.event.ApplicationInitializedEvent;
+import io.jans.service.cdi.event.LdapConfigurationReload;
+import io.jans.service.cdi.util.CdiUtil;
+import io.jans.service.custom.script.CustomScriptManager;
+import io.jans.service.logger.LoggerService;
+import io.jans.service.metric.inject.ReportMetric;
+import io.jans.service.timer.QuartzSchedulerManager;
+import io.jans.util.StringHelper;
+import io.jans.util.security.SecurityProviderUtility;
+import io.jans.util.security.StringEncrypter;
+import io.jans.util.security.StringEncrypter.EncryptionException;
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.BeforeDestroyed;
+import jakarta.enterprise.context.Initialized;
+import jakarta.enterprise.event.Event;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.inject.Instance;
+import jakarta.enterprise.inject.Produces;
+import jakarta.enterprise.inject.spi.BeanManager;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import jakarta.servlet.ServletContext;
+
+/**
+ * 
+ * Cache refresh server initializer
+ * @author Yuriy Movchan
+ * @version May 12, 2020
+ */
+@ApplicationScoped
+public class AppInitializer {
+
+	@Inject
+	private Logger log;
+
+	@Inject
+	private BeanManager beanManager;
+
+	@Inject
+	private Event<ApplicationInitializedEvent> eventApplicationInitialized;
+
+	@Inject
+	@Named(ApplicationFactory.PERSISTENCE_ENTRY_MANAGER_NAME)
+	private Instance<PersistenceEntryManager> persistenceEntryManagerInstance;
+
+	@Inject
+	@Named(ApplicationFactory.PERSISTENCE_ENTRY_MANAGER_NAME)
+	@ReportMetric
+	private Instance<PersistenceEntryManager> persistenceMetricEntryManagerInstance;
+
+	@Inject
+	private ApplicationFactory applicationFactory;
+
+	@Inject
+	private Instance<EncryptionService> encryptionServiceInstance;
+
+	@Inject
+	private PythonService pythonService;
+
+	@Inject
+	private MetricService metricService;
+
+	@Inject
+	private CustomScriptManager customScriptManager;
+
+	@Inject
+	private ConfigurationFactory configurationFactory;
+
+	@Inject
+	private CleanerTimer cleanerTimer;
+
+	@Inject
+	private QuartzSchedulerManager quartzSchedulerManager;
+
+	@Inject
+	private LoggerService loggerService;
+
+	@Inject
+	private CacheRefreshTimer cacheRefreshTimer;
+
+	@PostConstruct
+	public void createApplicationComponents() {
+		try {
+			SecurityProviderUtility.installBCProvider();
+		} catch (ClassCastException ex) {
+			log.error("Failed to install BC provider properly");
+		}
+	}
+
+	public void applicationInitialized(@Observes @Initialized(ApplicationScoped.class) Object init) {
+		log.debug("Initializing application services");
+
+		// Resteasy config - Turn off the default patch filter
```

Comment:
We can remove this because endpoints should be moved to API config

Reply:
done